.evergreen/install-dependencies.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+install_extension ()
+{
+   # Workaround to get PECL running on PHP 7.0
+   export PHP_PEAR_PHP_BIN=${PHP_PATH}/bin/php
+   export PHP_PEAR_INSTALL_DIR=${PHP_PATH}/lib/php
+
+   rm -f ${PHP_PATH}/lib/php.ini
+
+   if [ "x${DRIVER_BRANCH}" != "x" ] || [ "x${DRIVER_REPO}" != "x" ]; then
+      CLONE_REPO=${DRIVER_REPO:-https://github.com/mongodb/mongo-php-driver}
+      CHECKOUT_BRANCH=${DRIVER_BRANCH:-master}
+
+      echo "Compiling driver branch ${CHECKOUT_BRANCH} from repository ${CLONE_REPO}"
+
+      mkdir -p /tmp/compile
+      rm -rf /tmp/compile/mongo-php-driver
+      git clone ${CLONE_REPO} /tmp/compile/mongo-php-driver
+      cd /tmp/compile/mongo-php-driver
+
+      git checkout ${CHECKOUT_BRANCH}
+      git submodule update --init
+      phpize
+      ./configure --enable-mongodb-developer-flags
+      make all -j20 > /dev/null
+      make install
+
+      cd ${PROJECT_DIRECTORY}
+   elif [ "x${DRIVER_VERSION}" != "x" ]; then
+      echo "Installing driver version ${DRIVER_VERSION} from PECL"
+      pecl install -f mongodb-${DRIVER_VERSION}
+   else
+      echo "Installing latest driver version from PECL"
+      pecl install -f mongodb
+   fi
+
+   sudo cp ${PROJECT_DIRECTORY}/.evergreen/config/php.ini ${PHP_PATH}/lib/php.ini
```

Comment:
Would it make more sense to _append_ our `php.ini` to the existing one, or is there nothing of value in the original `php.ini`?

I looked at https://github.com/10gen/mongo-php-toolchain but I don't see any reference to `php.ini` at all. I dunno if that's something we might want to change down the line, as I happen to use the following for my local PHP builds:

```
date.timezone = "America/New_York"
display_errors = On
display_startup_errors = On
error_reporting = -1
log_errors = On
memory_limit = -1
report_memleaks = On
report_zend_debug = 0
short_open_tag = Off
```

Obviously `date.timezone` isn't relevant, but I think the other settings could be helpful to ensure PHP doesn't suppress memory leaks and such by default.

Reply:
I haven't focused on improving the process just yet, especially since testing in these branches is difficult as they are not tested on evergreen anyways. We could do the following:
* Move a sensible default configuration to mongo-php-toolchain to have it in place by default, and
* set `php_ini` in the pecl configuration so that `pecl install` enables the extension.

With this, we can remove this workaround, first in master to confirm that everything is working correctly, followed by a backport to these branches. WDYT?

Reply:
Fine to punt on this. I left a comment on the issue referring back to this thread for context.